dist/index.html

```diff
@@ -0,0 +1,67 @@
+  <title>Full Restaurant</title>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@500&family=Open+Sans:wght@300;400;600;700&display=swap"
+    rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+  <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+  <div id="carouselExampleCaptions" class="carousel slide" data-bs-ride="carousel">
+    <ol class="carousel-indicators w-50 mx-auto">
+      <li data-bs-target="#carouselExampleCaptions" data-bs-slide-to="0" class="active"></li>
+      <li data-bs-target="#carouselExampleCaptions" data-bs-slide-to="1"></li>
+      <li data-bs-target="#carouselExampleCaptions" data-bs-slide-to="2"></li>
+      <li data-bs-target="#carouselExampleCaptions" data-bs-slide-to="3"></li>
+    </ol>
+    <div class="carousel-inner">
+      <div class="carousel-item active">
+        <img src="../assets/food/breakfast.jpg" class="d-block food w-50 mx-auto" alt="food">
+        <div class="carousel-caption d-none d-md-block w-50 mx-auto">
+          <h5>Breakfast Special</h5>
+          <p>Nulla vitae elit libero, a pharetra augue mollis interdum.</p>
+        </div>
+      </div>
+      <div class="carousel-item">
+        <img src="../assets/food/lunch.jpg" class="d-block food w-50 mx-auto" alt="food">
+        <div class="carousel-caption d-none d-md-block w-50 mx-auto">
+          <h5>Lunch Spicy Special</h5>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </div>
+      </div>
+      <div class="carousel-item">
+        <img src="../assets/food/dinner.jpg" class="d-block food w-50 mx-auto" alt="food">
+        <div class="carousel-caption d-none d-md-block w-50 mx-auto">
+          <h5>Dinner Steak Special</h5>
+          <p>Praesent commodo cursus magna, vel scelerisque nisl consectetur.</p>
+        </div>
+      </div>
+      <div class="carousel-item">
+        <img src="../assets/food/dessert.jpg" class="d-block food w-50 mx-auto" alt="food">
+        <div class="carousel-caption d-none d-md-block w-50 mx-auto">
+          <h5>Dreamy Cream</h5>
+          <p>Praesent commodo cursus magna, vel scelerisque nisl consectetur.</p>
+        </div>
+      </div>
+    </div>
+    <a class="carousel-control-prev" href="#carouselExampleCaptions" role="button" data-bs-slide="prev">
+      <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+      <span class="visually-hidden">Previous</span>
+    </a>
+    <a class="carousel-control-next" href="#carouselExampleCaptions" role="button" data-bs-slide="next">
+      <span class="carousel-control-next-icon" aria-hidden="true"></span>
+      <span class="visually-hidden">Next</span>
+    </a>
+  </div>
```

Comment:
Well done @jcy2704 👍 

According to the instruction on `Odin` page, we should only have `<div id="content"></div>` inside the `body` tag. 

![image](https://user-images.githubusercontent.com/31897434/88849163-66288600-d1e1-11ea-8414-32df71189e88.png)

https://www.theodinproject.com/courses/javascript/lessons/restaurant-page